projects/schematics/src/shared/utils/dependency-utils_spec.ts

```diff
@@ -79,7 +79,13 @@ describe('dependency-util', () => {
         CDC_FEATURE_NAME,
       ]);
 
-      expect(result).toEqual([SPARTACUS_USER, SPARTACUS_ASM, SPARTACUS_CDC]);
+      // TODO: Double-check this.
+      expect(result).toEqual([
```

Comment:
Why are schematics unit tests updated in the scope of this PR?
Perhaps clarifying this is the purpose of the todo.